src/plugin.json

```diff
@@ -26,7 +26,7 @@
       "name": "Grafana Labs",
       "url": "https://grafana.com"
     },
-    "keywords": ["datasource", "athena", "aws", "cloud provider", "logs", "sql"],
+    "keywords": ["datasource", "athena", "aws", "amazon", "cloud provider", "logs", "sql"],
```

Comment:
Not sure if I should create a separate release for just this change as well as wether or not to cut release as part of these changes or in a subsequent PR 🤔

Reply:
they can probably wait, this ticket was in the backlog